backend/src/main/java/quarano/department/web/TrackedCaseController.java

```diff
@@ -327,14 +342,22 @@
 				.build();
 	}
 
-	private Stream<TrackedCaseContactSummary> createSummaries(TrackedCase trackedCase) {
```

Comment:
Let's avoid unrelated renames.

Reply:
What do you mean? 
I think the renaming is good to make the sense of the method more concrete compared to the new method createDiaryEntrySummaries.

Reply:
Yes, could you please clarify? I don't get what you mean with unrelated names.

Reply:
Ahh, that makes a lot of sense then. The context wasn't quite obvious to me. Thanks for elaborating.